CHANGELOG.md

```diff
@@ -4,7 +4,8 @@ Releases
 v1.20.0-dev (unreleased)
 --------------------
 
-- No changes yet.
+-   Added the `http.Interceptor` inbound option which allows the
```

Comment:
Nit:

    http: Add `http.Interceptor` option to inbounds, which allows intercepting
    incoming HTTP requests for that inbound.